src/main/java/Tasks/Deadline.java

```diff
@@ -0,0 +1,63 @@
+package Tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline task in a task management application.
+ * A deadline task includes a description and a deadline date and time.
+ * It extends the Task class by adding a deadline-specific time attribute.
+ * The deadline time is managed using the LocalDateTime class.
+ * The input time format is d/M/yyyy HHmm (e.g., 2/12/2019 1800).
+ * If an invalid date format is provided, an error message is displayed.
+ *
+ * @author Muhammad Rizki Bayuaji
+ */
```

Comment:
Perhaps you can omit some of the details here since you already explained it in more detail in the related method?